packages/Chem/src/analysis/chem-space.ts

```diff
@@ -6,14 +6,19 @@ import {chemGetFingerprints} from '../chem-searches';
 //import {getRdKitWebRoot} from '../chem-common-rdkit';
 import {Coordinates} from '@datagrok-libraries/utils/src/type-declarations';
 import {createDimensinalityReducingWorker,} from '@datagrok-libraries/ml/src/workers/dimensionality-reducing-worker-creator';
-import {MetricDataTypes} from '@datagrok-libraries/ml/src/string-measure';
+import {BitArrayMetrics, isBitArrayMetric, StringMetrics} from '@datagrok-libraries/ml/src/string-measure';
 import {Fingerprint} from "../utils/chem-common";
+import BitArray from '@datagrok-libraries/utils/src/bit-array';
 
 export async function chemSpace(table: DG.DataFrame, molColumn: DG.Column, methodName: string, similarityMetric: string) {
   let fpColumn = molColumn.toList();
-  if (MetricDataTypes['BitArray'].includes(similarityMetric))
+  let coordinates: Coordinates;
+  if (isBitArrayMetric(similarityMetric)) {
     fpColumn = await chemGetFingerprints(molColumn, Fingerprint.Morgan);
-  const coordinates = await createDimensinalityReducingWorker(fpColumn, methodName, similarityMetric) as Coordinates;
+    coordinates = await createDimensinalityReducingWorker({data: fpColumn as BitArray[], metric: similarityMetric as BitArrayMetrics}, methodName) as Coordinates;
+  } else {
```

Comment:
A bit confusing that Chem can require other than BitArray metrics. Is that correct at all?